CHANGELOG.md

```diff
@@ -5,12 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [0.19.0]
 
 ### Changed
 
 - `x-ccf-tx-view` and `x-ccf-tx-seqno` response headers have been removed, and replaced with `x-ms-ccf-transaction-id`. This includes both original fields, separated by a single `.`. Historical queries using `ccf::historical::adapter` should also pass a single combined `x-ms-ccf-transaction-id` header (#2257).
 - Node unique identifier is the hex-encoded string of the SHA-256 digest of the node's identity public key, which is also used as the node's quote report data (#2241).
+- The receipt interface has changed, `/app/receipt?commit=23` is replaced by `/app/receipt?transaction_id=2.23`. Receipt fetching is now implemented as a historical query, which means that the first reponse(s) may be a 202 with a Retry-After header. Receipts are now structured JSON, and `/app/receipt/verify` has been removed in favour of an [offline verification sample]().
```

Comment:
There's still a TODO at the end of this line, and we should also say:
- `get_receipt_for_seqno_v1` has been removed and should now be implemented through the historical query system
- Constructor for `UserFrontend`/`XXXEndpointRegistry` etc have changed, and should now be passed `AbstractNodeContext`

Reply:
Fixed now